src/ghhops-server-py/.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+    "files.exclude": {
+        "**/__pycache__": true
+    },
+    "python.pythonPath": "/Users/ein/.local/share/virtualenvs/ghhops-server-py-aHT-T4wI/bin/python",
```

Comment:
I'm not very familiar with this settings file. It has a path for your computer in it. Is this important?

Reply:
Removed!